bulletin/index.html

```diff
@@ -203,6 +203,26 @@ <h2 class="featured-article__title"> Engineering Robust Learning Environments Th
 	  </div>
 	</section>
 
+	<section>
+		<div class="featured-article__wrapper comments-acknowledge__wrapper">
+			<div class="comments-acknowledge__section">
+				<div class="comments-acknowledge__section--heading"> 
+					<h2>Comments &</h2>
+					<h2>Acknowledgments</h2>
```

Comment:
This is not proper HTMl because a screen reader or a crawler will read it as two `h2`. The correct way is to wrap the content that you want to break to a new line using an inline element, such as a `span`. 

```
<h2>Comments & Acknowledgements</span></h2>
```

Is there a design benefit to wrapping the text?

Reply:
This was corrected in more recent commits.  I was gaining a visual idea.